README.md

```diff
@@ -1,4 +1,4 @@
-# [Beta] Expo Eject
+# Expo Eject
 
 Using the The Expo Development CLI to Creates Xcode and Android Studio projects for your app.
```

Comment:
Use the Expo Development CLI to create Xcode and Android Studio projects for your app.